.github/workflows/niployments.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Upload to NIployments register
+      uses: NIAEFEUP/push-to-niployments@main
+      with:
+        project_name: nimentas-sasup
+        repository_name: website
+        docker_dockerfile: Dockerfile
+        docker_context: .
+        docker_target: prod
```

Comment:
This option is added by https://github.com/NIAEFEUP/push-to-niployments/pull/1.

Reply:
Why did we go with this approach instead of our usual `Dockerfile` for dev and `Dockerfile-prod` for prod? Any benefit?

Reply:
I understand that stages allow build parallelization but that makes me think we should stick to one single scheme throughout the organization itself. This _feels_ awkward.

Reply:
@thePeras @Ruben38Esteves food for thought

Reply:
From what I've seen, the recommended approach is to use a single Dockerfile for all services that require one, but with different build targets. This means dev, prod, but also if you need to change something in the postgres image, you should use the same Dockerfile, with different build targets. I think the reason for this is that it allows for reuse of repeated code. I personally like this approach.

Either way, this approach is used not because I wanted to (either way, I do), but because this is a fork of ttoino/ementas, which uses that approach. As a fork, it is in our interest to minimize the number of changes relative to the upstream repository, as to ease maintenance requirements.

Reply:
> @thePeras @Ruben38Esteves food for thought
Wrong Ruben @Naapperas :laughing: I think you meant this one @rubuy-74

Reply:
I agree with @limwa, it's technically not our code, we just forked it to make it more maintainable in the future without the attention from @ttoino. Because of this, I think it's fine to maintain the `Dockerfile` as-is, in this case.

Reply:
Can I mark this conversation as resolved?

Reply:
Understood, resolving the conversation.